README.md

```diff
@@ -60,6 +60,18 @@ last-branches -n 100
 - Node.js >= 16
 - A git repository (local branches are listed)
 
+## Build into a single Javascript file to be executed without `node_modules`
+
+Run the following command:
+
+```bash
+npm run build
+```
+
+This will compile the Javascript source code with dev dependencies in `node_modules`. This will allow you to execute the Javascript file in a separate directly without `node_modules` or global dependencies present.
+
+This can be useful in a dotfiles configuration folder with an alias that executes the Javascript file with node.
+
```

Comment:
so, what would be the full flow for a new customer who just wants to have `last-branches` command to be available in his command line?

The one proposed so far is `npm install -g last-branches-cli` and then it is there.

What would be the approach here for someone to do it?

Reply:
My dotfiles directory looks like this:
<img width="343" height="297" alt="image" src="https://github.com/user-attachments/assets/2e682eff-3b69-4723-8bc6-05370ee2b297" />

And in my zshrc file, i have:

`export PATH="$PATH:/Users/acherk/dotfiles/bin"`
`alias last-branches='node ./bin/last-branches.js'`

Reply:
so, do you want compiled version to be uploaded as well?
describe how would you be getting `last-branches.js`. I asked for full flow description for a new user.
what do you imageine?

Reply:
> so, do you want compiled version to be uploaded as well?

You could if you wanted. `npm run build` compiles it into `dist` which is currently gitignored.